core/esmf-aspect-model-document-generators/src/main/java/org/eclipse/esmf/aspectmodel/generator/jsonschema/AspectModelJsonSchemaVisitor.java

```diff
@@ -671,4 +672,8 @@ private ObjectNode addDescription( final ObjectNode node, final NamedElement des
 
       return node;
    }
+
+   private void addXSammAspectModelUrn( final ObjectNode node, final NamedElement describedElement ) {
+      node.put( "x-samm-aspect-model-urn", describedElement.getAspectModelUrn().map( Object::toString ).orElse( "" ) );
```

Comment:
I think it could happen that the NamedElement's AspectModelUrn is empty, in that case no `x-samm-aspect-model-urn` should be added IMO. Maybe like this:

```suggestion
      describedElement.getAspectModelUrn().ifPresent( urn -> node.put( "x-samm-aspect-model-urn", urn.toString() );
```

Reply:
Hm, got it.
I expect, if we don't have urn, we don't need to make this field in the spec. 

Okay,
I am going to fix this,

Thanks,
Best Regards!